### PR TITLE
Add colours to dels and ins

### PR DESCRIPTION
**Pull request by raissanorth**
image

Asked @clarkepaul for some further instructions, but haven't received an answer yet.

### CODE REVIEW

File: src/Forms/DiffTransformation.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function transform(FormField $field)
                 Diff::compareHTML($field->Value(), isset($this->data[$name]) ? $this->data[$name] : '')
             );
 
-            $diffField->addExtraClass($field->extraClass());
+            $diffField->addExtraClass($field->extraClass() . "history-viewer__version-detail-diff");
```
Comment (robbieaverill) [inline src/Forms/DiffTransformation.php]: Please add a space at the start of the quoted string, or add it with a separate call to `addExtraClass`

File: client/src/components/HistoryViewer/HistoryViewer.scss [hunk 2]
```diff
@@ -171,6 +171,27 @@
...
+    padding: .192rem;
+  }
+
+  ins {
+    background-color: $green;
+    color: white;
+    border-radius: .192 rem;
+    padding: .192 rem;
+  }
+}
```
Comment (robbieaverill) [inline client/src/components/HistoryViewer/HistoryViewer.scss]: Can you please run this through a linter?

File: client/src/components/HistoryViewer/HistoryViewer.scss [hunk 3]
```diff
@@ -171,6 +171,28 @@
...
+    color: white;
+    border-radius: .192rem;
+    padding: .192rem;
+  }
+
+  ins {
+    background-color: $green;
+    color: white;
+    border-radius: .192 rem;
+    padding: .192 rem;
```
Comment (robbieaverill) [inline client/src/components/HistoryViewer/HistoryViewer.scss]: not sure if the linter is picking this up, but you shouldn't have spaces between the number and the unit, also please remove the empty line on 175

Comment (raissanorth) [reply client/src/components/HistoryViewer/HistoryViewer.scss]: Neither `yarn lint` nor `yarn build` did...  But I have refactored that code anyway in the latest commit.

**Comment by raissanorth** [comment]
Updated styling:

image

**Pull request merged**